tensorhive/authorization.py

```diff
@@ -24,10 +24,10 @@ def add_claims_to_access_token(current_user_id):
         try:
             current_user = User.get(current_user_id)
             roles = current_user.role_names
-        except Exception as e:
+        except Exception as _:  # NOQA: F841
```

Comment:
After investigating all your changes I see it's very common and seemingly frustrating...
Replacing `except Exception as _:` or `except Exception as e:` (when e is not used) with:
```python
except Exception:
```
should do the job (I don't get any errors with this)

Reply:
fixed in the following commit